public/js/main-template.js

```diff
@@ -12,93 +12,124 @@ $(document).ready(function () {
+
+    if (event != "") {
+        if (event.currentTarget.classList.contains("PannelIcon")) {
+            if (document.getElementsByClassName("RightPanel2")[0].classList.contains("Active")) {
+                document.getElementsByClassName("PannelIcon2")[0].classList.remove("Active");
+                event.currentTarget.parentElement.classList.remove("RightPanel2Active");
+                document.getElementsByClassName("RightPanel2")[0].classList.remove("Active");
+            }
+            event.currentTarget.classList.toggle("Active");
+            event.currentTarget.parentElement.classList.toggle("RightPanel1Active");
+            document.getElementsByClassName("RightPanel1")[0].classList.toggle("Active");
+
+        }
+        if (event.currentTarget.classList.contains("PannelIcon2")) {
+            if (document.getElementsByClassName("RightPanel1")[0].classList.contains("Active")) {
+                document.getElementsByClassName("PannelIcon")[0].classList.remove("Active");
+                event.currentTarget.parentElement.classList.remove("RightPanel1Active");
+                document.getElementsByClassName("RightPanel1")[0].classList.remove("Active");
+            }
+            event.currentTarget.classList.toggle("Active");
+            event.currentTarget.parentElement.classList.toggle("RightPanel2Active");
+            document.getElementsByClassName("RightPanel2")[0].classList.toggle("Active");
+
+        }
+        if (event.currentTarget.classList.contains("LeftPannelIcon1")) {
+            event.currentTarget.classList.toggle("Active");
+            event.currentTarget.parentElement.classList.toggle("AsideBarLeftActive");
+        }
     }
-    if (!$(".AsideBarInstallRight").hasClass("RightPanel2Active")) {
-        addClasses([rightMenuBtn], "display-block");
-        removeClasses([rightMenuBtn], "display-none");
+    if (page == "install") {
+        document.getElementsByClassName("LeftPannelIcon1")[0].classList.add("Active");
+        document.getElementsByClassName("AsideBarLeft")[0].classList.add("AsideBarLeftActive");
+        document.getElementsByClassName("PannelIcon2")[0].classList.add("Active");
+        document.getElementsByClassName("AsideBarRight")[0].classList.add("RightPanel2Active");
+        document.getElementsByClassName("RightPanel2")[0].classList.add("Active");
+        document.getElementsByClassName("RightPanel1")[0].classList.remove("Active");
+        document.getElementsByClassName("AsideBarRight")[0].classList.remove("RightPanel1Active");
+        document.getElementsByClassName("PannelIcon")[0].classList.remove("Active");
+
     }
-    else {
-        removeClasses([rightMenuBtn], "display-block");
-        addClasses([rightMenuBtn], "display-none");
+    if (page == "field") {
+        document.getElementsByClassName("LeftPannelIcon1")[0].classList.remove("Active");
+        document.getElementsByClassName("AsideBarLeft")[0].classList.remove("AsideBarLeftActive");

```

Comment:
we have already a method in which you pass an element and a class which you want to remove so it will do for us. Just check that is you can use it here. same we have for addClass too.